jbpm-wb-common/jbpm-wb-data-filters-client/src/main/java/org/jbpm/workbench/df/client/list/base/DataSetQueryHelper.java

```diff
@@ -164,6 +164,15 @@ public Date getColumnDateValue(DataSet currentDataSet,String columnId, int index
         }
     }
 
+    public Boolean getColumnBooleanValue(DataSet currentDataSet,
+                                         String columnId,
+                                         int index) {
+        try {
+            return Boolean.valueOf((String) currentDataSet.getColumnById(columnId).getValues().get(index));
+        } catch (Exception e) {
+            return false;
```

Comment:
I'm concerned this could lead to inconsistent data showing as "false". Perhaps we should use Optional as a return so an invalid state can be better represented?